demos/virtual-paging.html

```diff
@@ -1,15 +1,16 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset=" utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="apple-mobile-web-app-capable" content="yes" />
-    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
-    <meta name="viewport" content="width = device-width, minimal-ui, initial-scale = 1, user-scalable = no" />
-    <meta name="apple-mobile-web-app-title" content="Datagrid">
-    <title>Datagrid - Paging</title>
-    <base href="/">
-    <style>
+
+<head>
```

Comment:
it looks like it did the tabbing thing again, would it help you if i implemented html lint/hint to the project?